test/inductor/test_codecache.py

```diff
@@ -1,6 +1,7 @@
 # Owner(s): ["module: inductor"]
```

Comment:
Style question - Is it pytorch style to not write explicit tests for the trion_bundler and maybe triton_heuristics files?

Reply:
All the tests I have seen so far have been end to end tests with counters to verify data. I could write more if you want but I'm just adhering to what currently exists

Reply:
Could we test the corner and errors cases in triton_bundler? I.e. begin_compile called twice, collect called twice, disk reads + writes throw errors?

Maybe even writes from two threads to make sure stuff is thread safe?